src/color/colormaker.ts

```diff
@@ -46,10 +46,16 @@ export const ScaleDefaultParameters = {
 }
 export type ScaleParameters = typeof ScaleDefaultParameters
 
+export interface ColorData {
+  atomData?: { [key: number]: number }
```

Comment:
This type definition is an inlined keyed object, when we are using number arrays.
I think it would be better typed by the following
```typescript
export interface ColorData {
    atomData?: number[],
    bondData?: number[]
}
```

Reply:
Also agreed (I was trying to keep it general for cases where you want to only supply data for a subset of atoms - i.e. something like: `atomData:  {0: 0.5, 55: 2.3, 148: 1.1}` but this is probably just premature optimization - I think under the hood sparse arrays are stored fairly efficiently in JS).